interface/docker_build_interface.sh

```diff
@@ -1,17 +1,37 @@
+script_directory=`dirname "${BASH_SOURCE[0]}"  | xargs realpath`
```

Comment:
I guess we don't have an official style guide for Bash scripts, but the extra space before the `|` bothers me a bit... we don't need that, right?

Reply:
Ah for sure - that extra space wasn't intentional - nice catch!